### PR TITLE
CAMEL-11762 - Add Karaf Feature for ASN.1 DataFormat

### PR DESCRIPTION
**Pull request by onders86**

### CODE REVIEW

File: tests/camel-itest-karaf/src/test/java/org/apache/camel/itest/karaf/CamelASN1Test.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.ops4j.pax.exam.junit.PaxExam;
+
+@RunWith(PaxExam.class)
+public class CamelASN1Test extends BaseKarafTest {
+
+    public static final String COMPONENT = extractName(CamelASN1Test.class);
+
+    @Test
+    public void test() throws Exception {
+        testDataFormat("asn1");
```
Comment (dmvolod) [inline tests/camel-itest-karaf/src/test/java/org/apache/camel/itest/karaf/CamelASN1Test.java]: This looks like testDataFormat(COMPONENT);
Please see other tests for sample

File: platforms/karaf/features/src/main/resources/features.xml [hunk 2]
```diff
@@ -114,6 +114,12 @@
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.java-apns/${java-apns-bundle-version}</bundle>
     <bundle>mvn:org.apache.camel/camel-apns/${project.version}</bundle>
   </feature>
+  <feature name='camel-asn1' version='${project.version}' resolver='(obr)' start-level='50'>
+    <feature version='${project.version}'>camel-core</feature>
```
Comment (dmvolod) [inline platforms/karaf/features/src/main/resources/features.xml]: Looks like the format a little bit incorrect, please put <feature and <bundle on the same XML level

File: tests/camel-itest-karaf/src/test/java/org/apache/camel/itest/karaf/CamelASN1Test.java (hunk 1 above)
Comment (onders86) [reply tests/camel-itest-karaf/src/test/java/org/apache/camel/itest/karaf/CamelASN1Test.java]: Please see

https://github.com/apache/camel/blob/master/tests/camel-itest-karaf/src/test/java/org/apache/camel/itest/karaf/CamelYamlTest.java#L26

File: platforms/karaf/features/src/main/resources/features.xml (hunk 2 above)
Comment (onders86) [reply platforms/karaf/features/src/main/resources/features.xml]: thanks.

**Comment by onders86** [comment]
Closed #1929

**Pull request closed without merging**